server/src/models/Review.js

```diff
@@ -26,6 +26,14 @@ class Review extends Model {
                     from: "reviews.movieId",
                     to: "movies.id"
                 }
+            },
+            User: {
```

Comment:
associations should be camelCase (lower cased)

Even though it has been merged in, for consistency with other associations the above `Movie` association should be lowercase `movie`. In the `relationMappings` we are defining "association methods" to use with $relatedQuery

https://vincit.github.io/objection.js/api/model/static-properties.html#static-relationmappings